### ISSUE TITLE
Which species does TCRMatch work with?

### ISSUE THREAD

**Issue by canuckafar** [IEDB/TCRMatch#36]
Thank you for providing TCRMatch for others to use. I wanted to know if it can be used for non-human species. For example, many researchers work with mice for TCR sequencing experiments. Could TCRMatch be used in the analysis of such data?

**Comment by acrinklaw** [comment]
We only measured the performance on human sequences, so I cannot confidently say that the results will be of value so I cannot formally recommend you use it in this way.

As for my personal opinion though...I have a hunch that the algorithm can be extended to non-human species. The underlying algorithm is pretty simple, just comparing k-mers and calculating scores with BLOSUM-62. The algorithm is so generalized that the original author of it (see paper [here](https://arxiv.org/pdf/1205.6031v2.pdf) used it to predict HLA peptide binding affinities and call HLA serotypes via clustering based on score. All we did was take the initial work and apply it to TCR specificities.

If you end up using it, you'd just swap out the data file with your own custom TSV that matches the format of "IEDB_data.tsv" in the data directory. Let me know how it goes if you try it out! Definitely should be benchmarked with some sort of control to ensure it actually has some predictive power.
